datafangst.md

```diff
@@ -2,13 +2,21 @@
 
 ## Leveranser 2021
 
+### Datafangst 2021-1.2.0 (29. januar)
+* Ny geometri med multilinje, multiflate elller multipunkt blir kun godtatt når dette tillatt i henhold til Datakatalogen
+* Rettet en feil som gjørde at det ikke var mulig å koble en datter fra NVDB til mor uten at datter samtidig hadde andre endringer
```

Comment:
gjørde -> gjorde

Reply:
Takk for at du fant disse, var tydeligvis ikke helt fokusert da jeg skrev...